dom-querying/main.js

```diff
@@ -0,0 +1,18 @@
+const h1 = document.querySelector('h1');
+
+console.log(h1);
+console.dir(h1);
```

Comment:
just a reminder to label your console.log statements, ex: `console.log('heading', h1)`